src/engraving/dom/engravingitem.cpp

```diff
@@ -2374,6 +2376,60 @@ EngravingItem::LayoutData* EngravingItem::mutldata()
     return m_layoutData;
 }
 
+const RectF& EngravingItem::LayoutData::bbox(LD_ACCESS mode) const
+{
+    if (m_item->type() == ElementType::REST) {
+        int k = -1;
```

Comment:
Would be good to remove/change this soon (but possibly in a next PR) to avoid the compiler warning

Reply:
oops

Reply:
removed